webapp/channels/src/components/profile_popover/profile_popover_controller.tsx

```diff
@@ -139,6 +144,7 @@ export function ProfilePopoverController<TriggerComponentType = HTMLSpanElement>
                                 ref={refs.setFloating}
                                 style={combinedFloatingStyles}
                                 className={classNames('user-profile-popover', A11yClassNames.POPUP)}
+                                aria-label={`${props.username}'s profile popover`}
```

Comment:
A couple things here:
1. This should be translatable. You can use React Intl's `useIntl()` to get access to `intl` and then `intl.formatMessage` for that. You'll also have to run `npm run i18n-extract -w channels` to update the translation files with the new translated text.
2. Since the `username` prop is optional, we need to handle that case as well. I think it'd be fine to make the aria label just "profile popover" when `username` is empty or missing